beancount.el

```diff
@@ -178,6 +178,9 @@ from the open directive for the relevant account."
 (defconst beancount-date-regexp "[0-9]\\{4\\}[-/][0-9]\\{2\\}[-/][0-9]\\{2\\}"
   "A regular expression to match dates.")
 
+(defconst beancount-timestamp-indent-regexp
```

Comment:
I agree that `beancount-timestap-indent-regexp` is not the best name. Something like `beancount-timestamp-relaxed-indentation-regexp` or similar may be more clear to a reader.

My purpose for using a `defconst` here was because I use this regexp in multiple places, and IMO minimizing the number of times one repeats themselves in code is a good thing.

My assumption was also that `defconst` evaluates the expression only once at file load time, which would be a minor (and, admittedly, probably completely negligible) optimization when the expression is used in multiple different places throughout the file and in functions that get repeatedly executed throughout the duration of the program.